docs/usage.txt

```diff
@@ -63,6 +63,33 @@ Result:
         <link rel="preload" href="/static/CACHE/js/d01466eb4fc6.js" as="script" />
 
 
+
+ES5 nomodule & ES6 type="module"
+
+Adding the ``nomodule`` attribute will generate be passed attribute to the script tag for the compressed resource in the template.
+Adding the ``module`` parameter will generate the script tag with type="module" for the compressed resource in the template:
```

Comment:
I'd suggest clarifying that `nomodule` needs to go into the script html tag, and `module` needs to go into the compress templatetag.